.github/workflows/test-db-ops.yaml

```
@@ -25,13 +27,135 @@ on:
+      - name: Checkout wf-data-migration
+        uses: actions/checkout@v4
+        with:
+          repository: ballerine-io/wf-data-migration
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+          ref: ${{ github.event_name == 'repository_dispatch' && github.event.client_payload.ref || inputs.environment }}
+          fetch-depth: 1
+          path: services/workflows-service/prisma/data-migrations
+      
+      - name: Cache Docker layers
+        id: cache
+        uses: actions/cache@v2
```

actionlint

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)